server/src/app.ts

```diff
@@ -8,12 +8,14 @@ import { router } from './router';
 import dotenv from 'dotenv';
 import jwt from 'jsonwebtoken';
 import { dbCheck } from './db/ClientController';
+import session from 'express-session';
 
 const cookieParser = require('cookie-parser');
 const bodyParser = require('body-parser');
 
 dotenv.config();
 const jwtTokenSecret = process.env.JWT_TKN_SECRET;
+const sessionTokenSecret = process.env.SESSION_SECRET;
```

Comment:
Is `SESSION_SECRET` something that should be included in `/server/.env.base` or otherwise documented?

Reply:
Yes this should be included. And we should ungracefully fall over if it is not defined with no option for a backup val.

Reply:
I've added that a first step under "Deployment instructions". What kind of value should be set? Does it need to match with a value from some other system, or is it just a secret internal to the app?

It would be good if the `.env.base` file were updated in the repo to include this new variable.

Reply:
this is done